report_qweb_encrypt/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Creu Blanca
+# Copyright 2020 Ecosoft Co., Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Report Qweb Encrypt",
+    "summary": "Allow to encrypt qweb pdfs",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Creu Blanca,Ecosoft,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/reporting-engine",
+    "depends": [
+        "web",
+    ],
+    "data": [
+        "views/ir_actions_report.xml",
+    ],
+    "assets": {
+        "web.assets_backend": [
+            "report_qweb_encrypt/static/src/js/report/action_manager_report.esm.js",
+            "report_qweb_encrypt/static/src/js/report/encrypt_dialog.xml",
```

Comment:
You're placing a XML file under the js directory.

I think what you should do instead is removing the js directory and place both files inside report, because, starting from OWL structure, files are not placed according to their languages but according to their purpose.

Reply:
I think this makes sense and is consistent with other modules' structures, I've corrected this, thanks!